### PR TITLE
Add src_dir to load path

### LINKED ISSUE #153: app/lib should be in load path
I've seen a couple reports of load path weirdness. Let's make sure that `app/lib` (or more specifically the value of `Pakyow::Config.app.src_dir`) is present in the load path when the app is prepared.

### PR DESCRIPTION
**Pull request by bryanp**
Brings some sanity to `require` statements and fixes #153.

### CODE REVIEW

**Comment by bryanp** [comment]
@iamvery care to look this one over before I merge?

File: pakyow-core/spec/unit/helpers/running_spec.rb [hunk 1]
```diff
@@ -1,12 +1,13 @@
-require_relative '../../spec_helper'
+# require_relative '../../spec_helper'
```
Comment (iamvery) [inline pakyow-core/spec/unit/helpers/running_spec.rb]: Any reason to keep this around?

Comment (iamvery) [inline pakyow-core/spec/unit/helpers/running_spec.rb]: Why don't you need to require the spec helper anymore?

File: pakyow-core/spec/unit/helpers/running_spec.rb [hunk 2]
```diff
@@ -130,6 +138,11 @@ def self.run(*)
       mock.prepare(*envs)
     end
 
+    it 'adds src_dir to load path' do
+      mock.prepare(*envs)
+      expect($:).to include(mock.src_dir)
```
Comment (iamvery) [inline pakyow-core/spec/unit/helpers/running_spec.rb]: Maybe use `$LOAD_PATH` instad of `$:`. magic--

File: pakyow-core/lib/core/helpers/running.rb [hunk 3]
```diff
@@ -24,6 +24,9 @@ def prepare(*env_or_envs)
         # load each block from middleware stack
         load_middleware
 
+        # add app/lib to load path
+        $:.unshift config.app.src_dir
```
Comment (iamvery) [inline pakyow-core/lib/core/helpers/running.rb]: Maybe use `$LOAD_PATH` instad of `$:`. magic--

**Comment by iamvery** [comment]
Is the `app` directory itself already in the load path when pakyow starts?

File: pakyow-core/lib/core/helpers/running.rb (hunk 3 above)
Comment (bryanp) [inline pakyow-core/lib/core/helpers/running.rb]: Good call; fixed

File: pakyow-core/spec/unit/helpers/running_spec.rb (hunk 2 above)
Comment (bryanp) [inline pakyow-core/spec/unit/helpers/running_spec.rb]: Fixed

File: pakyow-core/spec/unit/helpers/running_spec.rb (hunk 1 above)
Comment (bryanp) [inline pakyow-core/spec/unit/helpers/running_spec.rb]: That was a mistake; fixed

**Comment by bryanp** [comment]
@iamvery Just saw your question about the `app` directory. No, it's not in the load path when pakyow starts. Curious to hear if you have something else you're thinking about :grin:

**Comment by iamvery** [comment]
not really yet. i'll need to actually make a thing :sweat_smile:

**Comment by bryanp** [comment]
Ha, cool. Going to merge this in the meantime since it's an improvement.

**Comment by iamvery** [comment]
:100:

**Pull request merged**